build.gradle

```diff
@@ -205,4 +213,7 @@ task generateClient(dependsOn: [deleteFiles], type: GenerateTask) {
             scmDeveloperConnection: "scm:git:git@github.com:hwslabs/iam-java-client.git",
             scmUrl: "https://github.com/hwslabs/iam-java-client.git"
     ]
+    additionalProperties = [
+            library: "retrofit2"
```

Comment:
We are using coroutines in every client, any specific cause why we are not using it here?